apps/cookbook/src/app/examples/avatar-example/examples/colors.ts

```diff
@@ -51,7 +51,8 @@ const config = {
   selector: config.selector,
   template: config.template,
   styleUrls: ['./avatar-examples.shared.scss'],
-  // tslint:disable-next-line: no-host-metadata-property
+
+  // eslint-disable-next-line @angular-eslint/no-host-metadata-property
```

Comment:
We could take our own medicine here and actually do the refactor to `@HostBinding` already, since all these `no-host-metadata-property` errors are cookbook-related, so the risk is quite low. 

Thoughts?

Reply:
Yeah I agree! 😄